amlogic.cmake

```diff
@@ -91,6 +91,11 @@ if (SKY_BUILD)
     add_definitions (-DSKY_BUILD)
 endif()
 
+if (SUPPRESS_MAINTENANCE)
```

Comment:
Pls remove this SUPPRESS_MAINTENANCE build flag to simplify the code. If we cannot read activation status from authservice then we can default to existing behavior.

Reply:
This is specific to Platco & Hisense, so we are restricting this to these platforms based on the distro feature. Hence activation check is only for the aforesaid platforms, rest of the platforms we don't enable it.